### ISSUE TITLE
mismatch in example and explanation in tutorial in the docs

### ISSUE THREAD

**Issue by rajsing** [tomchristie/django-rest-framework#2493]
https://github.com/tomchristie/django-rest-framework/blob/master/docs/tutorial/1-serialization.md#creating-a-serializer-class

The section on creating a serializer class has a line saying 

"The {'base_template': 'textarea.html'} flag above is equivelent to using widget=widgets.Textarea ...."
The example actually however has 
`code = serializers.CharField(style={'type': 'textarea'})`

**Comment by jpadilla** [comment]
I think that's meant to say:

```python
code = serializers.CharField(style={'base_template': 'textarea.html'})
```

**Comment by tomchristie** [comment]
Sounds right @jpadilla

**Comment by jpadilla** [comment]
@tomchristie PR coming up

**Comment by jpadilla** [comment]
@rajsing Thanks for reporting this. Fixed in #2497.
